CMakeLists.txt

```diff
@@ -227,7 +227,7 @@ IF (EXPAT_FOUND)
     LIST(APPEND TST_INCLUDES ${EXPAT_INCLUDE_DIRS})
     LIST(APPEND TST_LIBS ${EXPAT_LIBRARIES})
 ELSE (EXPAT_FOUND)
-    MESSAGE(WARNING "We can't find expat for XML crap")
+    MESSAGE(FATAL_ERROR "We can't find expat for XML crap")
```

Comment:
since we're updating can we make the message a little nicer?

Reply:
LOL. Yes, we can.

Reply:
Fixed.